content/blog/__trashed-2/index.ar.md

```
@@ -19,7 +19,7 @@ lang: "ar"
 
 نعلن اليوم عن خطة ترجمان, احد اهم مشاريع مجتمع أسس للمساهمة في التَّرْجَمَةً و دعم اللغة العربية في البرمجيات الحرة والمفتوحة.
 
-![](aosus-torjoman-1024x576-f6e96c63-2.jpg)
+![](aosus-torjoman-1024x576-f6e96c63.jpg)
```

markdownlint-cli2 (0.22.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)